query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/relations/rel_names.rs

```diff
@@ -0,0 +1,59 @@
+use query_engine_tests::*;
+
+// We were creating a child_a record instead of child_b.
+// https://github.com/prisma/prisma/issues/14696
+//
+// This was due to a relation resolution logic issue.
+#[test_suite]
+mod rel_names {
```

Comment:
The idea with this test was to check that the anomaly where we were writing to the wrong table doesn't exist anymore. It may not be in the right place.